plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -448,6 +538,8 @@ static void ffmpeg_source_update(void *data, obs_data_t *settings)
 	s->close_when_inactive =
 		obs_data_get_bool(settings, "close_when_inactive");
 
+	s->audio_track = obs_data_get_int(settings, "audio_track");
```

Comment:
Related to the above nit, `video_track` could be added here too, allowing very advanced users to select the video track using obs-websocket.